pom.xml

```diff
@@ -12,7 +12,7 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<tomcat.version>7.0.50</tomcat.version>
+		<tomcat.version>7.0.96</tomcat.version>
```

Comment:
could this be the latest version which you upraded to?

Reply:
HerbertYiga thanks for your comment. I have worked on it